.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 # Generate defintions out of schemas by running `yarn generate`
 lib/generated
+*/.DS_Store
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏you might want to ignore .DS_Store files globally: https://dev.to/lpheller/quickly-ignore-dsstore-files-globally-ecd